doc/cephfs/nfs.rst

```diff
@@ -79,3 +79,234 @@ Current limitations
+
+Minikube Setup
+--------------
+
+This setup is done on Fedora 30 and following package versions were used::
+
+        minikube v1.4.0, Kubernetes v1.16.0, Docker 18.09.9, git 2.21.0, kubernetes-client 1.15.2-1, docker-common 1.13.1-68 and qemu-kvm 3.1.1-2.
+
+Installation::
+
+        dnf install kubernetes-client docker-common git qemu-kvm
+        cd /tmp
+        curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && chmod +x minikube
+        curl -LO https://storage.googleapis.com/minikube/releases/latest/docker-machine-driver-kvm2 && chmod +x docker-machine-driver-kvm2
+        cp minikube docker-machine-driver-kvm2 /usr/local/bin
+
+.. note:: Place the minkube and docker executables in $PATH. Make sure $HOME has enough space as minikube requires about 25 GB.
+
+
+Deploy Minikube Cluster::
+
+        minikube start --vm-driver=kvm2
+
+.. note:: Add --memory and --cpus options to provide custom RAM and CPU values for the VM.
+
+List the running pods::
+
+        kubectl get pods --all-namespaces
+
+Rook Setup And Cluster Deployment
+---------------------------------
+
+Clone the rook repository::
+
+        git clone https://github.com/rook/rook.git
+
+Deploy the Rook Operator::
+
+        cd cluster/examples/kubernetes/ceph
+        kubectl create -f common.yaml
+        kubectl create -f operator.yaml
+
+.. note:: Nautilus release or latest Ceph image should be used.
+
+Before proceding check if the pods are running::
+
+        kubectl -n rook-ceph get pod
+
+
+.. note::
+        For troubleshooting on any pod use::
+
+                kubectl describe -n rook-ceph pod <pod-name>
+
+In cluster-test.yaml file set **allowMultiplePerNode** to **true** under mon. As we
+will be using 3 monitors. If using minikube cluster change the **dataDirHostPath**
+to **/data/rook**. This is to make sure data persists across reboots.
+
+Deploy the ceph cluster::
+
+        kubectl create -f cluster-test.yaml
+
+To interact with Ceph Daemons, let's deploy toolbox::
+
+        kubectl create -f ./toolbox.yaml
+
+Exec into the rook-ceph-tools pod::
+
+        kubectl -n rook-ceph exec -it $(kubectl -n rook-ceph get pod -l "app=rook-ceph-tools" -o jsonpath='{.items[0].metadata.name}') bash
+
+Check if you have one Ceph monitor, manager, OSD running and cluster is healthy::
+
+        [root@minikube /]# ceph -s
+           cluster:
+                id:     3a30f44c-a9ce-4c26-9f25-cc6fd23128d0
+                health: HEALTH_OK
+
+           services:
+                mon: 1 daemons, quorum a (age 14m)
+                mgr: a(active, since 13m)
+                osd: 1 osds: 1 up (since 13m), 1 in (since 13m)
+
+           data:
+                pools:   0 pools, 0 pgs
+                objects: 0 objects, 0 B
+                usage:   5.0 GiB used, 11 GiB / 16 GiB avail
+                pgs:
+
+Scale out monitors::
+
+        [root@minikube /]# ceph orchestrator mon update 3
+        Updated mon count to 3
```

Comment:
I know this was part of @jtlayton 's original blogs. Why do we need to increase the number of mons for the purposes of this document?

Reply:
Yup, it is not required here. I have updated the doc.